src/main/ts/components/__tests__/CleanAsYouCode-test.tsx

```diff
@@ -86,11 +100,32 @@ it('should properly render the projection curve', async () => {
 
   expect(await screen.findByText('cayc.chart.title')).toBeInTheDocument();
 
-  const projection = ui.projectionCurve.get().getAttribute('d');
+  expect(ui.projectedIssueCount.get()).toHaveTextContent('0.6k issues');
+
+  await selectEvent.select(ui.periodSelect.get(), 'cayc.chart.title.period_option.6 months');
 
-  await selectEvent.select(ui.select.get(), 'cayc.chart.title.period_option.6 months');
+  expect(ui.projectedIssueCount.get()).toHaveTextContent('0.2k issues');
+
+  const history1 = [];
+  for (let i = 0; i < 36; i++) {
+    history1.push(getHistoryEntry(i, 2));
+  }
+  jest.mocked(getIssues).mockResolvedValueOnce(history1);
+
+  const projectSelect = byRole('combobox').getAll().at(0)!;
+
+  await selectEvent.select(projectSelect, 'Project 2');
```

Comment:
This will work once you'll have fixed the accessibility issue.

```suggestion
  await selectEvent.select(ui.projectSelect.get(), 'Project 2');
```